sp_mocks/imsharer/backend/index.js

```diff
@@ -4,7 +4,7 @@ const fs = require('fs');
 const layer8 = require('layer8_middleware');
 
 const app = express();
-const port = 6001;
+const port = 8000;
```

Comment:
I would rather we leave this as 6001 and instead test the capability of the proxy on ports like this other than 8000